docs-2.0/1.introduction/3.nebula-graph-architecture/2.meta-service.md

```diff
@@ -0,0 +1,56 @@
+# Meta Service
+
+This topic describes the architecture and functions of the Meta Service.
+
+## The architecture of the Meta Service
+
+The architecture of the Meta Service is as follows.
+
+![The architecture of the Meta Service](https://docs-cdn.nebula-graph.com.cn/docs-2.0/1.introduction/2.nebula-graph-architecture/meta-architecture.png)
+
+The Meta Service is supported by the nebula-metad processes. You can deploy the nebula-metad processes on a single machine or multiple machines. All the nebula-metad processes form a Raft-based cluster, with one process as the leader and the others as the followers.
+
+The leader is elected by all the nebula-metad processes in the cluster, and only the leader can provide service to the clients and other components of the Nebula Graph cluster. The followers run in a stand-by way and each has a data replication of the leader. Once the leader fails, one of the followers will be elected as the new leader.
+
+## Functions of the Meta Service
+
+### Manages user accounts
+
+The Meta Service stores the information of user accounts and the privileges granted to the accounts. When the clients send queries to the Graph Service through an account, the Graph Service checks the account information and whether the account has the right privileges to execute the queries or not.
+
+For more information on Nebula Graph access control, see [Authentication and authorization](TODO) (TODO).
+
+### Manages partitions
+
+The Meta Service stores and manages the locations of the storage partitions and helps balance the partitions.
+
+### Monitors service status
+
+The Meta Service receives heartbeats from the servers in the cluster to monitor the connections in case of failure.
+
+### Manages graph spaces
+
+Nebula Graph supports multiple graph spaces. Data stored in different graph spaces are securely isolated. The Meta Service stores the metadata of all graph spaces and tracks the changes of them, such as adding or dropping a graph space and modifying graph space configuration.
+
+### Manages schema information
+
+Nebula Graph is a strong-typed graph database. Its schema structures are tags (i.e., the vertex types), edge types, tag properties, and edge type properties.
+
+The Meta Service stores the schema information. Besides, it performs the addition, modification, and deletion of the schema structures, and logs the versions of them.
+
+For more information on Nebula Graph schema, see [Nebula Graph schema and data modeling](../2.nebula-graph-schema-and-data-modeling) (TODO).
+
+### Manages TTL-based data eviction
+
+The Meta Service provides automatic data eviction and space reclamation based on TTL (time to live) options for Nebula Graph.
```

Comment:
I’m not sure if TTL supports space reclamation.
And I prefer TTL to TTL  option.